podcasts/Bookmarks/Player/BookmarksPlayerTab.swift

```diff
@@ -14,11 +14,16 @@ struct BookmarksPlayerTab: View {
 
     var body: some View {
         VStack(spacing: 0) {
-            headerView
-            divider
+            if viewModel.items.isEmpty {
+                BookmarksEmptyStateView(style: .playerStyle)
+                Spacer()
+            } else {
+                headerView
+                divider
 
-            actionBarView {
-                scrollView
+                actionBarView {
+                    scrollView
+                }
```

Comment:
Maybe extract these views to their own properties? Then we can do something like:

```
viewModel.items.isEmpty ? emptyView : listView
```

Reply:
Nice, done here: 9e5df15fae53130dda03c4a3c1fade25c7eaa8db